package.json

```diff
@@ -1,19 +1,41 @@
 {
-    "name": "xunit-file",
-    "description": "Basically the same reporter as mocha's xunit reporter, but writes the output in a file.",
-    "version": "0.0.6",
-    "author": {
-        "name": "Matthias Jahn",
-        "email": "matthias.jahn@peerigon.com"
+  "name": "xunit-file",
```

Comment:
`npm install --save` reformatted this, let me know if you want me to un-format it.